scripts/builtin/img_cutout_linearized.dml

```diff
@@ -0,0 +1,74 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+# Image Cutout function replaces a rectangular section of an image with a constant value.
+#
+# INPUT:
+# ---------------------------------------------------------------------------------------------
+# img_in      Input images as linearized 2D matrix with top left corner at [1, 1]
+# x           Column index of the top left corner of the rectangle (starting at 1)
+# y           Row index of the top left corner of the rectangle (starting at 1)
+# width       Width of the rectangle (must be positive)
+# height      Height of the rectangle (must be positive)
+# fill_value  The value to set for the rectangle
+# s_cols     Width of a single image
+# s_rows     Height of a single image
+# ---------------------------------------------------------------------------------------------
+#
+# OUTPUT:
+# ------------------------------------------------------------------------------------------
+# img_out Output images as linearized 2D matrix with top left corner at [1, 1]
```

Comment:
for not obvious reasons we need to spaces between the img_out and the beginning of the documentation